setup.py

```diff
@@ -20,6 +24,27 @@
 with open("README.rst", "rb", encoding="utf-8") as f:
     readme = f.read()
 
+# Ensure Bootstrap
+# Important: when you update the Bootstrap version, make sure to also update the
+# harcoded included files and versions in:
```

Comment:
hardcoded is missing a "d" --- ha I kept subconsciously inserting the "d" and couldn't figure out the issue

Reply:
It's not caught by Codespell, ugh.